### PR TITLE
[Analytics] refactor analytics url

### PR DESCRIPTION
**Pull request by ja-odur**
#### Summary of Changes (What does this PR do?)
- refactor analytics urls

#### Is this change ready to hit production in its current state?
- [ ] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [ ] I consider this code done
- [ ] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Fetch this branch locally
* Create a python virtual env (if none exists) python3 -m venv ./virtual_env_name`
* Activate the virtual env `source ./virtual_env_name/bin/activate`
* Install requirements `pip install -r requirements.txt`
* Start application `flask run`
* Test all analytics endpoints. ensure the endpoints work and they contain the prefix `/api/v1/analytics/`.

#### What are the relevant tickets?
- [PLAT-498](https://airqoteam.atlassian.net/browse/PLAT-498)

### CODE REVIEW

File: src/analytics/controllers/helpers.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def get_and_save_daily_measurements():
     return jsonify({'response': 'all daily measurements saved'}), 200
 
 
-@analytics_app.route('/api/v1/device/measurements/daily/update', methods =['GET'])
+@analytics_app.route('/api/v1/device/analytics/measurements/daily/update', methods =['GET'])
```
Comment (danielogen) [inline src/analytics/controllers/helpers.py:39]: looks like "analytics" should come before "device" here

**Review by danielogen — CHANGES_REQUESTED** [review]
This is looking great already...just one correction to be made. Awesome work!!

File: src/analytics/controllers/helpers.py (hunk 1 above)
Comment (ja-odur) [reply src/analytics/controllers/helpers.py]: @danielogen thank you for the keen observation. This has been fixed.

**Review by danielogen — APPROVED** [review]
LGTM

**Review by Baalmart — COMMENTED** [review]
Thanks @ja-odur 
LGTM, I suggest that we make the necessary updates to the Analytics documentation here: https://docs.airqo.net/airqo-platform-api/analytics cc: @sserurich

**Comment by ja-odur** [comment]
> Thanks @ja-odur
> LGTM, I suggest that we make the necessary updates to the Analytics documentation here: https://docs.airqo.net/airqo-platform-api/analytics cc: @sserurich

Will work on this immediately after the PR merge

File: src/analytics/controllers/dashboard.py [hunk 2]
```diff
@@ -215,7 +215,7 @@ def download_customised_data():
             return final_data
 
 
-@dashboard_bp.route('/api/v1/dashboard/customisedchart/random/chartone', methods=['GET'])
+@dashboard_bp.route('/api/v1/analytics/dashboard/customisedchart/random/chartone', methods=['GET'])
```
Comment (sserurich) [inline src/analytics/controllers/dashboard.py:218]: @ja-odur , not sure weather this is still being consumed anywhere. may be it can go away

**Review by sserurich — APPROVED** [review]
looks good for me, the endpoints have tested out are working perfectly. Have not tested the endpoints to do with reports.

**Review by Baalmart — APPROVED** [review]
LGTM

**Pull request merged**